### PR TITLE
Expand edit label functionality

### PR DESCRIPTION
**Pull request by blurpesec**
[ch6098]

### Description
Add edit label functionality into more places.

### ToDo
- [x] Add edit label functionality to tx receipt panel

### CODE REVIEW

File: src/features/SwapAssets/components/ConfirmSwap.tsx [hunk 1]
```diff
@@ -57,8 +57,15 @@ export default function ConfirmSwap({
         toAmount={toAmount.toString(10)}
       />
       <FromToAccount
-        from={{ address: account.address, label: account.label }}
-        to={{ address: account.address, label: account.label }}
+        fromAccount={{
+          address: account.address,
+          addressBookEntry: { ...account, notes: '', network: account.networkId }
```
Comment (FrederikBolding) [inline src/features/SwapAssets/components/ConfirmSwap.tsx:62]: Won't this possibly remove notes for existing accounts if updating the label?

Comment (FrederikBolding) [reply src/features/SwapAssets/components/ConfirmSwap.tsx:62]: Same for below

File: src/components/TransactionFlow/ConfirmTransaction.tsx [hunk 2]
```diff
@@ -200,15 +198,15 @@ export const ConfirmTransactionUI = ({
...
+          addressBookEntry: senderContact
         }}
-        to={{
-          address: (receiverAddress || 'Unknown') as never,
-          label: recipientLabel
+        toAccount={{
+          address: receiverAddress,
+          addressBookEntry: recipientContact
         }}
-        displayToAddress={txType !== ITxType.DEFIZAP && txType !== ITxType.PURCHASE_MEMBERSHIP}
```
[the remark this answers is not in the document]
Comment (blurpesec) [reply src/components/TransactionFlow/ConfirmTransaction.tsx:211]: Nice catch :D

File: src/components/TransactionFlow/displays/FromToAccount.tsx [hunk 3]
```diff
@@ -87,10 +105,16 @@ export default function FromToAccount({ from, to, displayToAddress = true }: Pro
...
+            />
           </AddressWrapper>
         </AddressContainer>
       )}
     </Addresses>
   );
-}
+};
+
+export default FromToAccount;
```
[the remark this answers is not in the document]
Comment (blurpesec) [reply src/components/TransactionFlow/displays/FromToAccount.tsx:120]: oversight :dagger:

File: src/components/Account.tsx [hunk 4]
```diff
@@ -18,8 +21,11 @@ const Content = styled.div`
 
 const Title = styled(Typography)`
   display: inline;
-  font-size: ${scale(0.5)};
+  font-size: ${FONT_SIZE.BASE};
```
[the remark this answers is not in the document]
Comment (blurpesec) [reply src/components/Account.tsx:24]: Yeah, I figured we wanted all of the Account components to be standard component styles (bold editable label, truncated address, etc)

File: src/features/SwapAssets/components/ConfirmSwap.tsx (hunk 1 above)
Comment (blurpesec) [reply src/features/SwapAssets/components/ConfirmSwap.tsx]: i made a change that should cover this as long as im not dumb xD. please take a look!

File: src/components/Account.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (FrederikBolding) [reply src/components/Account.tsx:24]: Can you verify this with design or someone else?

[the remark this answers is not in the document]
Comment (FrederikBolding) [reply src/components/Account.tsx:24]: It seems like an elaborate choice to have it at a larger font?

File: src/components/EditableAccountLabel.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  networkId: NetworkId;
+}
+
+const SWrapper = styled.span`
+  @media (min-width: ${BREAK_POINTS.SCREEN_SM}) {
+    font-weight: bold;
+  }
+`;
+
+const EditableAccountLabel = ({ addressBookEntry, address, networkId }: Props) => {
```
Comment (FrederikBolding) [inline src/components/EditableAccountLabel.tsx:23]: Can we add a test for this component?

File: src/features/SwapAssets/components/ConfirmSwap.tsx [hunk 6]
```diff
@@ -57,8 +62,23 @@ export default function ConfirmSwap({
         toAmount={toAmount.toString(10)}
       />
       <FromToAccount
-        from={{ address: account.address, label: account.label }}
-        to={{ address: account.address, label: account.label }}
+        fromAccount={{
+          address: account.address,
+          addressBookEntry: {
```
Comment (FrederikBolding) [inline src/features/SwapAssets/components/ConfirmSwap.tsx:67]: Can we not just use `fromAddressBookEntry`?

(Same for below)

**Pull request merged**